src/main/java/com/avispl/symphony/dal/communicator/lg/lcd/LgLCDDevice.java

```diff
@@ -1,369 +1,402 @@
+/*
+ * Copyright (c) 2021 AVI-SPL, Inc. All Rights Reserved.
+ */
 package com.avispl.symphony.dal.communicator.lg.lcd;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.util.CollectionUtils;
+
 import com.avispl.symphony.api.dal.control.Controller;
 import com.avispl.symphony.api.dal.dto.control.ControllableProperty;
 import com.avispl.symphony.api.dal.dto.monitor.ExtendedStatistics;
 import com.avispl.symphony.api.dal.dto.monitor.Statistics;
 import com.avispl.symphony.api.dal.monitor.Monitorable;
 import com.avispl.symphony.dal.communicator.SocketCommunicator;
-
-import java.util.*;
-
-import static com.avispl.symphony.dal.communicator.lg.lcd.LgLCDConstants.*;
-
+import com.avispl.symphony.dal.communicator.lg.lcd.LgLCDConstants.commandNames;
+import com.avispl.symphony.dal.communicator.lg.lcd.LgLCDConstants.controlProperties;
+import com.avispl.symphony.dal.communicator.lg.lcd.LgLCDConstants.replyStatusNames;
+import com.avispl.symphony.dal.communicator.lg.lcd.LgLCDConstants.statisticsProperties;
+
+/**
+ * LG LCD Device Adapter
+ *
+ * Static Monitored Statistics
+ * <li>
+ * Input
+ * Power
+ * Fan
+ * SyncStatus
+ * </li>
+ *
+ * Historical Monitored Statistics
+ * <li> Temperature </li>
+ *
+ * @author Harry
+ * @since 1.2
```

Comment:
Please also add the @version on this class. And the @since for every method and member data that's added in to this version

Reply:
I get on nuclino that we add @since as an adapter version for the communicator class.

Reply:
I fixed it